docs/questions/native-editor/writing-sql.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: The SQL editor
+title: SQL editor
 redirect-from:
   - /docs/latest/users-guide/writing-sql
 ---
 
-# The SQL editor
```

Comment:
In the query builder section, we just call the corresponding article "Editor". Also articles are unnecessary in headlines imo